OpenTabletDriver/Environ/DriverInfo.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using OpenTabletDriver.Environ.Drivers;
+
+namespace OpenTabletDriver.Environ
```

Comment:
Just make the full namespace name
```suggestion
namespace OpenTabletDriver.Environment
```

Reply:
This makes it conflict with System.Environment on a lot of places.